recipe/meta.yaml

```diff
@@ -11,7 +11,8 @@ source:
 
 build:
   script: python setup.py install --single-version-externally-managed --record record.txt
-  number: 0
+  number: 1
+  noarch: python
```

Comment:
Interesting feature, indeed! Is this line enough for `conda-smithy` to render the feedstock correctly?

Reply:
Yep. First you change that line and then re-render.
(But I am having trouble with the entrypoint now :unamused:)